query.go

```diff
@@ -37,6 +37,7 @@ func (fb *Firebase) EndAt(value string) *Firebase {
 func (fb *Firebase) OrderBy(value string) *Firebase {
 	c := fb.copy()
 	if value != "" {
+		value = "\"" + value + "\""
```

Comment:
stylistic nit: could use backticks here to avoid the escapes

```go
value = `"` + value + `"`
```

Or, we could `import "fmt"` and use `fmt.Sprintf()`:

```go
value = fmt.Sprintf("%q", value)
```

The `fmt` option would be desirable if `value` might contain unsafe characters or characters that need to be escaped (such as a `"`).